src/Common/ProfileEvents.cpp

```diff
@@ -190,8 +190,13 @@
     \
     M(ReplicaPartialShutdown, "How many times Replicated table has to deinitialize its state due to session expiration in ZooKeeper. The state is reinitialized every time when ZooKeeper is available again.") \
     \
+    M(PrimaryKeyNotUsed, "Check if primary key is used in this query.") \
```

Comment:
The new ProfileEvents are orthogonal to the extension of `sytem.query_log`. Could we split either of these into a separate PR?

Reply:
ok